.github/workflows/auto-pr.yml

```
@@ -0,0 +1,4 @@
+components/ui/link-preview/
```

actionlint (1.7.4)

workflow is scalar node but mapping node is expected

(syntax-check)